license.rst

```diff
@@ -0,0 +1,72 @@
+==========================
+END USER LICENSE AGREEMENT
+==========================
+
+for Anaconda, Miniconda, and Conda
```

Comment:
The Anaconda license is different from the conda license. The conda license is the BSD license listed [here](https://github.com/conda/conda/blob/master/LICENSE.txt).